pkg/kubeadm/util/ipaddr.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeadmutil
+
+import (
+	"fmt"
+	"net"
+)
+
+func GetDefaultHostIP() (string, error) {
```

Comment:
Thanks, will look into that. I've been trying hard to find existing helpers, obviously forgot to look this one up... :)